src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -181,7 +181,7 @@
   /**
    * overload this if your service needs other environmental or dependencies to be ready
    */
-  protected boolean isReady = true;
+  protected boolean ready = true;
```

Comment:
Hmm.. it's a bigger refactor to do this, but  "ready" should probably be defaulted to "false" ..  And it's up to each service to decide when it's ready to set it to true.  (I think this is a bigger discussion that GroG should weigh in on.)

Reply:
to be, or not to be. @GROG will awnser the question.
I think it need to be false for services that implement it, true by default

Reply:
I changed my mind I prefer "ready" now :)